README.md

```diff
@@ -1,44 +1,67 @@
+[![Published on webcomponents.org](https://img.shields.io/badge/webcomponents.org-published-blue.svg)](https://webcomponents.org/element/@polymer/paper-input)
 
 ## &lt;paper-input&gt;
+`<paper-input>` is a single-line text field with Material Design styling.
 
-Material design: [Text fields](https://www.google.com/design/spec/components/text-fields.html)
+See: [Documentation](https://www.webcomponents.org/element/@polymer/paper-input),
+  [Demo](https://www.webcomponents.org/element/@polymer/paper-input/demo/demo/index.html).
 
-`<paper-input>` is a single-line text field with Material Design styling.
+## Usage
 
-<!---
+### Installation
 ```
-<custom-element-demo>
-  <template>
-    <script src="../webcomponentsjs/webcomponents-lite.js"></script>
-    <link rel="import" href="paper-input.html">
-    <link rel="import" href="../iron-icons/iron-icons.html">
-    <style>
-      paper-input {
-        max-width: 400px;
-        margin: auto;
-      }
-      iron-icon, div[suffix] {
-        color: hsl(0, 0%, 50%);
-        margin-right: 12px;
-      }
-    </style>
-    <next-code-block></next-code-block>
-  </template>
-</custom-element-demo>
+npm install --save @polymer/paper-input
 ```
--->
+
+### In an html file
 ```html
-<paper-input always-float-label label="Floating label"></paper-input>
-<paper-input label="username">
-  <iron-icon icon="mail" slot="prefix"></iron-icon>
-  <div slot="suffix">@email.com</div>
-</paper-input>
+<html>
+  <head>
+    <script type="module">
+      import '@polymer/paper-input/paper-input.js';
+    </script>
+  </head>
+  <body>
+    <paper-input always-float-label label="Floating label"></paper-input>
+  </body>
+</html>
+```
+### In a Polymer 3 element
+```js
+import {PolymerElement, html} from '@polymer/polymer';
+import '@polymer/paper-input/paper-input.js';
+
+class SampleElement extends PolymerElement {
+  static get template() {
+    return html`
+      <paper-input always-float-label label="Floating label"></paper-input>
+    `;
+  }
+}
+customElements.define('sample-element', SampleElement);
 ```
 
-### Changes in 2.0
-- Distribution is now done with slots, so things have changed because of that (before: <label>foo</label>, now <label slot="label">foo</label>; same for input, prefix and suffix)
-- Since type extensions are not available, any use of `<input is="iron-input">` has been replaced by `<iron-input><input></iron-input>`. See [iron-input](https://github.com/PolymerElements/iron-input) for more details about that change
-- `prevent-invalid-input` and `allowed-pattern` had to be always used together; deleted `prevent-invalid-input`, so that only `allowed-pattern` is needed
+## Contributing
+If you want to send a PR to this element, here are
+the instructions for running the tests and demo locally:
 
+### Installation
+```sh
+git clone https://github.com/PolymerElements/paper-input
+cd paper-input
+npm install
+npm install -g polymer-cli
```

Comment:
Maybe add that `npm install -g polymer-cli` is a prereq? or do it first, before the clone with a comment "make sure you have the polymer cli installed before you start"?